platform/macos/core-files.txt

```diff
@@ -0,0 +1,95 @@
+# This file is generated. Do not edit. Regenerate this with scripts/generate-cmake-files.js
```

Comment:
If the results of this script really have to be checked in for some reason, let’s add it as a Run Script build phase. I don’t think we’d get into a situation where one of these files needs to be listed but doesn’t need to be built before being committed. Perhaps it should be in the intermediate build products or build products directory so that it would be deleted when cleaning the project in Xcode.

Reply:
The point of this file is to have a standalone file that is independent of the Xcode project for use in other build systems when integrating Mapbox GL in a larger build environment. The `nitpick` build will complain if there's a mismatch between the Xcode project and the generated file as a reminder to run it.